suzieq/config/devconfig.yml

```diff
@@ -1,5 +1,5 @@
 service: devconfig
-period: 3600
+period: 30
```

Comment:
Is this a debug leftover? Why change this default?

Reply:
Ugh. Good catch, will fix before merging. Thanks.